jobs/dap-collector/dap_collector/main.py

```diff
@@ -0,0 +1,194 @@
+from google.cloud import bigquery
+import click
+import datetime
+import math
+import os
+import re
+import subprocess
+import time
+import typing
+
+HPKE_CONFIG = "MgAgAAEAAQAgjsCwHpkLTLxj1Y01O8FiidMmU1gGRd2Vnwv8ctXLRH8"
```

Comment:
HPKE_CONFIG can probably stay hardcoded because it should rarely change so we can just update it here if necessary. Updating this config would also require updating the private keys. Long-term, tasks will have to be configured some other way which is tbd.